transport-classes-io_uring/src/main/java/io/netty/channel/uring/AbstractIoUringStreamChannel.java

```diff
@@ -204,6 +205,17 @@ protected final void doRegister(ChannelPromise promise) {
         });
     }
 
+    @Override
+    protected Object filterOutboundMessage(Object msg) {
+        // Since we cannot use synchronous sendfile,
+        // the channel can only support DefaultFileRegion instead of FileRegion.
+        if (msg instanceof DefaultFileRegion) {
```

Comment:
Perhaps we should insert the verification of whether the current system supports splice here？
``` java
        if (msg instanceof DefaultFileRegion && IoUring.isIOUringSpliceSupported()) {
           //...
        }
```